client/src/modules/vouchers/templates/action.cell.html

```diff
@@ -39,6 +39,11 @@
         <span class="fa fa-file-text-o"></span> <span translate>REPORT.VIEW_VOUCHER</span>
       </a>
     </li>
+    <li ng-show="row.entity.type_id === 12 || row.entity.type_id === 13 || row.entity.type_id === 14">
+      <a data-method="view-stock-movements" ui-sref="stockMovements({ filters : [{ key : 'voucherReference', value : row.entity.reference, displayValue: row.entity.reference, cacheable:false }, { key : 'period', value : 'allTime', cacheable:false }]})" href>
```

Comment:
Could you move this and the `data-method="view-voucher"` up in the list?  It seems to be below the reverse record option and I can see users frequently mis-clicking.

![image](https://user-images.githubusercontent.com/896472/113264020-66c27080-92ca-11eb-9baa-732b4ab310c0.png)